tour_model_eval/viz_similarity_unlabeled.ipynb

```diff
@@ -30,7 +31,7 @@
     "    N = len(bars)\n",
     "    index = numpy.arange(N)\n",
     "    width = .2\n",
-    "    fig = plt.figure()\n",
+    "#     fig = plt.figure()\n",
```

Comment:
you can remove commented out code since you can always look it up from GitHub history if you need to go back.